reports/m1_proposal.md

```diff
@@ -11,14 +11,28 @@ In a few sentences, provide motivation for why you are creating a dashboard:
 
 ## Description of the data
 
-You are allowed to select any dataset you want for this project,
-as long as you have the license to use it publicly.
-Finding a suitable data set can take a lot of time and effort.
-You are therefore allowed to select one
-that you have worked with in a previous lab/project in MDS
-if you wish.
-
-
+We will be visualizing a dataset of approximately 18000 rows of Vancouver daily weather data queried via an open-mateo 
+API. More information about the data source can be explored here: https://open-meteo.com/en/docs.
+
+Within the query extraction code, we have specified a latitude and longitude to represent Vancouver as per below:
+VAN_LAT = 49.2497
+VAN_LONG = -123.1193
+The specific coordinate selected to present Vancouver is selected based on reference of this website 
+https://latitude.to/map/ca/canada/cities/vancouver. It is actually quite close to King Edward Canada Line station. 
+
+For the time period, we extracted data from 1/1/1974 to 3/15/2024, a period of 50 years. Each row of data contains 
+many variable of interests that could be helpful in identifying the noteable trend in Vancouver weather over this long 
+period. 
+
+Below are the list of variables, grouped into important factors affecting how we perceive weather.
+- Overall weather: "weather_code"
```

Comment:
I think you would borrow some description of variable from the webesite, such as, weather code: _Weather condition as a numeric code. Follow WMO weather interpretation codes. See table below for details. Weather code is calculated from cloud cover analysis, precipitation, snowfall, cape and gusts._ I think we need to give some variables examples.